lib/osf-components/addon/components/registries/read-only-contributors-list/template.hbs

```diff
@@ -0,0 +1,14 @@
+<div local-class='contributorContainer' ...attributes>
+    <ContributorList
+        @node={{this.node}}
+        @shouldTruncate={{false}}
+        @shouldLinkUsers={{true}}
+    />
+    <OsfLink
+        data-test-edit-contributors-link
+        data-analytics-name='edit_contributors'
```

Comment:
Our other analytics labels don't use snake_case; was this deliberate or should it be `Edit contributors`?